api/src/main/java/org/pmiops/workbench/mail/MailServiceImpl.java

```diff
@@ -69,29 +69,89 @@ public void sendBetaAccessRequestEmail(String userName) throws MessagingExceptio
   }
 
   @Override
-  public void sendWelcomeEmail(String contactEmail, String password, User user)
+  public void sendWelcomeEmail(final String contactEmail, final String password, final User user)
       throws MessagingException {
-    try {
-      InternetAddress email = new InternetAddress(contactEmail);
-      email.validate();
-    } catch (AddressException e) {
-      throw new MessagingException("Email: " + contactEmail + " is invalid.");
-    }
-    MandrillMessage msg = buildWelcomeMessage(contactEmail, password, user);
```

Comment:
There was a lot of duplication in the `buildXMessage` and `buildXHtml` methods, so I made this process more generic.  I'll need to add 3 more of these for RW-3661.